plugins/inputs/nvidia_smi/testdata/rtx-3080-v12.xml

```diff
@@ -0,0 +1,786 @@
+		<compute_mode>Default</compute_mode>
+		<utilization>
+			<gpu_util>0 %</gpu_util>
+			<memory_util>37 %</memory_util>
+			<encoder_util>0 %</encoder_util>
+			<decoder_util>0 %</decoder_util>
+			<jpeg_util>0 %</jpeg_util>
+			<ofa_util>0 %</ofa_util>
+		</utilization>
+		<encoder_stats>
+			<session_count>0</session_count>
+			<average_fps>0</average_fps>
+			<average_latency>0</average_latency>
+		</encoder_stats>
+		<fbc_stats>
+			<session_count>0</session_count>
+			<average_fps>0</average_fps>
+			<average_latency>0</average_latency>
+		</fbc_stats>
+		<ecc_mode>
+			<current_ecc>N/A</current_ecc>
+			<pending_ecc>N/A</pending_ecc>
+		</ecc_mode>
+		<ecc_errors>
+			<volatile>
+				<sram_correctable>N/A</sram_correctable>
+				<sram_uncorrectable>N/A</sram_uncorrectable>
+				<dram_correctable>N/A</dram_correctable>
+				<dram_uncorrectable>N/A</dram_uncorrectable>
+			</volatile>
+			<aggregate>
+				<sram_correctable>N/A</sram_correctable>
+				<sram_uncorrectable>N/A</sram_uncorrectable>
+				<dram_correctable>N/A</dram_correctable>
+				<dram_uncorrectable>N/A</dram_uncorrectable>
+			</aggregate>
+		</ecc_errors>
+		<retired_pages>
+			<multiple_single_bit_retirement>
+				<retired_count>N/A</retired_count>
+				<retired_pagelist>N/A</retired_pagelist>
+			</multiple_single_bit_retirement>
+			<double_bit_retirement>
+				<retired_count>N/A</retired_count>
+				<retired_pagelist>N/A</retired_pagelist>
+			</double_bit_retirement>
+			<pending_blacklist>N/A</pending_blacklist>
+			<pending_retirement>N/A</pending_retirement>
+		</retired_pages>
+		<remapped_rows>N/A</remapped_rows>
+		<temperature>
+			<gpu_temp>31 C</gpu_temp>
+			<gpu_temp_tlimit>N/A</gpu_temp_tlimit>
+			<gpu_temp_max_threshold>98 C</gpu_temp_max_threshold>
+			<gpu_temp_slow_threshold>95 C</gpu_temp_slow_threshold>
+			<gpu_temp_max_gpu_threshold>93 C</gpu_temp_max_gpu_threshold>
+			<gpu_target_temperature>91 C</gpu_target_temperature>
+			<memory_temp>N/A</memory_temp>
+			<gpu_temp_max_mem_threshold>N/A</gpu_temp_max_mem_threshold>
+		</temperature>
+		<supported_gpu_target_temp>
+			<gpu_target_temp_min>65 C</gpu_target_temp_min>
+			<gpu_target_temp_max>91 C</gpu_target_temp_max>
+		</supported_gpu_target_temp>
+		<gpu_power_readings>
+			<power_state>P8</power_state>
+			<power_draw>22.78 W</power_draw>
+			<current_power_limit>336.00 W</current_power_limit>
+			<requested_power_limit>336.00 W</requested_power_limit>
+			<default_power_limit>320.00 W</default_power_limit>
+			<min_power_limit>100.00 W</min_power_limit>
+			<max_power_limit>336.00 W</max_power_limit>
+		</gpu_power_readings>
+		<module_power_readings>
+			<power_state>P8</power_state>
+			<power_draw>N/A</power_draw>
+			<current_power_limit>N/A</current_power_limit>
+			<requested_power_limit>N/A</requested_power_limit>
+			<default_power_limit>N/A</default_power_limit>
+			<min_power_limit>N/A</min_power_limit>
+			<max_power_limit>N/A</max_power_limit>
+		</module_power_readings>
```

Comment:
I don't see this in the v12 parser. What do you think about adding the module power readings as well?

Reply:
Since this is a feature PR anyway, why not. ;-)

Reply:
Done.